src/commands/schema/ls.js

```diff
@@ -0,0 +1,44 @@
+const FaunaCommand = require('../../lib/fauna-command.js')
+const fetch = require('node-fetch')
+const { errorOut } = require('../../lib/misc.js')
+
+class ListSchemaCommand extends FaunaCommand {
+  async run() {
+    const log = this.log
+    const {
+      connectionOptions: { domain, port, scheme, secret },
+    } = await this.getClient()
+
+    return fetch(`${scheme}://${domain}:${port}/schema/1/files`, {
+      method: 'GET',
+      headers: { AUTHORIZATION: `Bearer ${secret}` },
+    })
+      .then(function (res) {
+        return res.json()
+      })
+      .then(function (json) {
+        if (json.files.length > 0) {
+          log('Schema files')
+          json.files.forEach(function (file) {
+            log(file.filename)
+          })
+        } else {
+          log('No schema files')
+        }
+      })
+      .catch(function (err) {
+        errorOut(err)
+      })
```

Comment:
style: js has `await` for this:
```suggestion
    try {
      const res = await fetch(`${scheme}://${domain}:${port}/schema/1/files`, {
        method: 'GET',
        headers: { AUTHORIZATION: `Bearer ${secret}` },
      });
      const json = await res.json();
      if (json.files.length > 0) {
        log('Schema files')
        json.files.forEach(function (file) {
          log(file.filename)
        })
      } else {
        log('No schema files')
      }
    } catch (e) {
      errorOut(e);
    }
```